packages/projection/package.json

```diff
@@ -0,0 +1,75 @@
+{
+  "name": "@cardano-sdk/projection",
+  "version": "0.0.1",
```

Comment:
```suggestion
  "version": "0.1.0",
```

Conventionally we've been starting at this version

Reply:
Updated to 0.1.0 (sorry should've made a fixup for this, but already did it when squashing the util-rxjs type util commit